ui/app/serializers/config-ui/message.js

```diff
@@ -3,22 +3,37 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { encodeString } from 'core/utils/b64';
+import { decodeString, encodeString } from 'core/utils/b64';
 import ApplicationSerializer from '../application';
 
 export default class MessageSerializer extends ApplicationSerializer {
   primaryKey = 'id';
 
+  normalizeResponse(store, primaryModelClass, payload, id, requestType) {
+    if (requestType === 'queryRecord') {
+      const transformed = {
+        ...payload.data,
+        message: decodeString(payload.data.message),
+        link_title: payload.data.link.title,
+        link_href: payload.data.link.href,
+      };
+      delete transformed.link;
+      return super.normalizeResponse(store, primaryModelClass, transformed, id, requestType);
+    }
+    return super.normalizeResponse(store, primaryModelClass, payload, id, requestType);
+  }
+
   serialize() {
     const json = super.serialize(...arguments);
     json.message = encodeString(json.message);
     json.link = {
-      title: json.link_title,
-      href: json.link_href,
+      title: json?.link_title || '',
+      href: json?.link_href || '',
     };
 
     delete json.link_title;
     delete json.link_href;
+    delete json.active;
 
```

Comment:
same comment here about deleting via the serialize false. Basically we use this for attributes the API does not want or expect on the response or payload